src/main/java/uk/gov/hmcts/reform/roleassignment/util/PersistenceUtil.java

```diff
@@ -150,4 +152,21 @@ public RoleAssignment convertEntityToRoleAssignment(RoleAssignmentEntity roleAss
                                 Collections.emptyList())
             .build();
     }
+
+    public HistoryEntity prepareHistoryEntityForPersistance(RoleAssignment roleAssignment, Request request) {
+        UUID roleAssignmentId = roleAssignment.getId();
+        UUID requestId = request.getId();
```

Comment:
Why we moved this method from Persistence Service to here?

Reply:
Hi Dharmendra,
Earlier, the persistance was happening through persistenceService.addHistoryEntity() method. Now, we have removed the persistance logic from old method and moved it outside loop. Current method "prepareHistoryEntityForPersistance" is only preparing the object and not doing any persist operation. Hence moved it from PersistabceService to PersistanceUtil.

Reply:
Ok Fine.